restli-server/src/test/java/com/linkedin/restli/internal/server/model/SampleResources.java

```diff
@@ -171,7 +178,7 @@ public String code()
     namespace = "com.linkedin.restli.internal.server.model",
     parent = ParentResource.class
   )
-  class TestResource extends CollectionResourceTemplate<String, EmptyRecord>
+  static class TestResource extends CollectionResourceTemplate<String, EmptyRecord>
```

Comment:
You make this change because you put these two new test classes into different packages, why
cannot we put them together in the same package as existing `TestRestLiApiBuilder` for easier discovery later?

Reply:
They are in the same package! The problem was that I was importing `static` in the new test file. Fixed that and reverted the static modifiers in `SampleResources`.